File: src/main/java/olivermakesco/de/refmagic/block/RefinedMagicOreBlock.java
package olivermakesco.de.refmagic.block;

import net.minecraft.block.OreBlock;
import net.minecraft.util.math.intprovider.UniformIntProvider;

public class RefinedMagicOreBlock extends OreBlock {
    public RefinedMagicOreBlock(Settings settings) {
        super(settings, UniformIntProvider.create(1, 2));
    }
}

File: src/main/kotlin/olivermakesco/de/refmagic/block/RefinedMagicOreBlock.kt
package olivermakesco.de.refmagic.block

import net.minecraft.block.Blocks
import net.minecraft.block.ExperienceDroppingBlock
import net.minecraft.util.math.intprovider.UniformIntProvider

class RefinedMagicOreBlock(settings: Settings) : ExperienceDroppingBlock(settings, UniformIntProvider.create(1, 2))